reviewrot/phabricatorstack.py

```diff
@@ -0,0 +1,251 @@
+import logging
+from phabricator import Phabricator
+import datetime
+from reviewrot.basereview import BaseService, BaseReview, LastComment
+
+log = logging.getLogger(__name__)
+
+
+class PhabricatorService(BaseService):
+    """
+        This class represents Phabricator Service for Review Rot.
+        The reference can be found here: https://www.phacility.com/phabricator/
+        This class takes in a list of repo_name (i.e. usernames) and then
+        attempts to find all open reviews where those users are responsible.
+        Currently this does not query based on  different repositories.
+    """
+    def request_reviews(self, host, token, repo_name=None, state_=None,
```

Comment:
If searching by 'repo' is not something that makes sense for phabricator then it would probably be a good idea to remove the argument from the method to avoid any possible confusion

Reply:
Agreed.  More comment on this below.